packages/hint-validate-set-cookie-header/src/hint.ts

```diff
@@ -340,17 +340,11 @@ export default class ValidateSetCookieHeaderHint implements IHint {
 
                 parsedSetCookie.resource = resource;
 
-                defaultValidators.every((defaultValidator) => {
-                    const messages: ValidationMessages = defaultValidator(parsedSetCookie);
-
-                    if (messages.length) {
-                        reportBatch(messages, codeLanguage, codeSnippet);
-
-                        return false;
-                    }
-
-                    return true;
+                const messages = defaultValidators.flatMap((defaultValidator) => {
```

Comment:
The `flatMap` usage threw me at first because we only have a single-depth array of validators (I missed they each return an array of messages). Not sure if a comment might help others here.

Unfortunately I think we're going to have to use something else because [according to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap), `flatMap` is only supported in Node 11+ (and we currently support Node 10+ in webhint). CI didn't catch this because we only run Node 12 on PRs currently.

You can probably fairly easily switch to using `flatMap` from Lodash if you want as we use Lodash in a number of other places. Just be sure to use the `import flatMap = require('lodash/flatMap');` syntax as that helps us avoid bundling all of Lodash with webhint's browser extension (we'd like to switch to ESM bundling with tree shaking in the future, but haven't gotten there yet).

Reply:
@antross I changed it to use `reduce`.